nav2_smac_planner/test/test_node_lattice.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2021, Samsung Research America
```

Comment:
This is probably your copyright header :wink:

Reply:
Done